### ISSUE TITLE
Node.js (API/Fullstack) Developer Positions @ Console Connect

### ISSUE THREAD

**Issue by iamkevinv** [BrisJS/meetups#197]
## Code in JS, Revolutionise the internet!

TLDR: Check out this incredible Node.JS API developer position: http://grnh.se/vg7tqf1 (all positions advertised on our greenhouse board).

Definitely one of the best opportunities around to join our great growing team, with a Silicon Valley & Brisbane based company, which retains its startup culture and passion for creating an incredible new internet service product - http://consoleconnect.com/

We'd especially like to hear from people who are interested in backend/API development with us, but we're always interested in all flavours of JS developers, whether you like developing with browserified node frontend JS, or all of the above, it would be great to see people with range.

We've been privileged to take on board some of our BrisJS members before ( older posts: #52 ), and are continually impressed with the skill and passion of Brisbane devs who are keen to make a great product together.

We're keen to find some more great team members, so please have a look at the official job posting http://grnh.se/vg7tqf1 and feel welcome to get in touch with me on this issue or at kvanderbeken@consoleconnect.com if you have any questions. 
That's the official public link, so we put a simple screening task on there to show you at least know JS, and stem the torrent of 1 click applicants from places like seek ;-). If you feel that's onerous you should let me know that too!

Look forward to fielding your questions!
-Kevin Vanderbeken
